src/app.py

```
@@ -138,16 +148,16 @@ def get_transcript(video_id: str = Query(..., description="YouTube video ID'si")
 
-        return {"summary_turkish": summary_in_turkish}
+        return {"summary": summary}
 
     except HTTPException as http_exc:
-        logger.error(f"HTTP hatası oluştu: {http_exc.detail}")
+        logger.error(f"HTTP error occurred: {http_exc.detail}")
         raise http_exc
     except Exception as e:
-        logger.error(f"Genel bir hata oluştu: {e}")
+        logger.error(f"General error occurred: {e}")
         raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)